### ISSUE TITLE
Does it work on Windows

### ISSUE THREAD

**Issue by PierrunoYT** [ex3ndr/llama-coder#31]
I saw Ollama does not support Windows yet

**Comment by Kevsnz** [comment]
Llama Coder extension works on Windows just fine. But you need to run Ollama in other environment - Linux or MacOS, and configure this extension to use that.

In Windows you could try WSL, Ollama apparently supports it [Ollama README](https://github.com/ollama/ollama#windows)

**Comment by leolivier** [comment]
You can install ollama on Windows using WSL2. See the ollama install on linux chapter which also applies to WSL2

**Comment by leolivier** [comment]
Ollama has now a windows version

**Comment by taylorn-ai** [comment]
I have Ollama and the Llama Coder installed on Windows. I can see Ollama starting up and working, but it doesn't work with the Llama Coder extension...

**Comment by leolivier** [comment]
Did you pull a model in Ollama?
In a windows terminal, run eg
ollama pull codellama

**Comment by TaylorNQT** [comment]
> Did you pull a model in Ollama?
> In a windows terminal, run eg
> ollama pull codellama

Yes I pulled the `codellama:7b-code-q4_K_M` image and it runs fine if I run from cmd.

**Comment by taylorn-ai** [comment]
> Did you pull a model in Ollama?
> In a windows terminal, run eg
> ollama pull codellama

Yes I pulled the `codellama:7b-code-q4_K_M` image and it runs fine if I run from cmd.
